packages/feedback-react/documentation/Example.tsx

```diff
@@ -12,6 +13,32 @@ export const Example = ({ boolValues }: ExampleComponentProps) => {
                 description="Hvor fornøyd er du med denne siden for å følge saken?"
                 onSubmit={console.info}
                 showTextArea={!boolValues?.["Uten tekst"]}
+                renderCustomSuccess={(props) => (
+                    <div>
+                        <SuccessMessage title="Tilbakemelding sendt!">
+                            {props.value === VERY_UNHAPPY && <>Det var trist!</>}
```

Comment:
not a big deal, men kunne deconstructe props

Reply:
Tenkte dette var tydeligere for å illustrere renderprop-bruken. Men ja, normalt ville jeg nok destructa, så kanskje ikke en strålende grunn.